src/public/media.ts

```diff
@@ -77,3 +81,233 @@ export function captureImage(callback: (error: SdkError, files: File[]) => void)
   const messageId = sendMessageRequestToParent('captureImage');
   GlobalVars.callbacks[messageId] = callback;
 }
+
+/**
+ * Media object returned by the select Media API
+ */
+export class Media extends File {
+  /**
+   * A preview of the file which is a lightweight representation.
+   * In case of images this will be a thumbnail in base64 encoding.
+   */
+  public preview: string;
+
+  /**
+   * Gets the media in chunks irrespecitve of size, these chunks are assembled and sent back to the webapp as file/blob
+   * @param callback returns blob of media
+   */
+  public getMedia(callback: (error: SdkError, blob: Blob) => void): void {
```

Comment:
media.getMedia() sounds better than getContent() or getBlob(). getContent() is not good because we have a member named content in Media object and getBlob() is kind of evident as the return param is blob itself. Also I am kind of pushing back on changing names unless must have, at this point of time because the name changes will have to be propagated in all the three platforms, in all design docs and wiki pages and test apps.